src/agent/solana/exporter.rs

```diff
@@ -213,6 +221,12 @@ pub struct Exporter {
     // Channel on which to send inflight transactions to the transaction monitor
     inflight_transactions_tx: Sender<Signature>,
 
+    /// Permissioned symbols as read by the oracle module
+    perm_price_rx: mpsc::Receiver<HashSet<Pubkey>>,
+
+    /// Symbols we are permissioned to publish
+    perm_price_accounts: HashSet<Pubkey>,
```

Comment:
maybe these permissions should go in `NetworkState`? I think that already contains network-specific information, which this is also.

Reply:
eh... maybe not. i see why you added this channel, because you want to get the information from the Oracle, which is already getting the other price account information. Maybe this is fine.